app/plugins/Wallet/js/buttons.js

```diff
@@ -25,8 +41,8 @@ function appendAddress(address) {
 
 // Add the new address
 addResultListener('new-address', function(result) {
-	notify('Address created!', 'created');
-	appendAddress(result.address);
+	notify('New address created', 'created');
+	appendAddress(result);
```

Comment:
this will add the new address even if it doesn't match the current search term. Adding addresses is a bit janky now anyway, so maybe I'll just revisit this later.